### DISCUSSION TITLE
Rename Collections

### DISCUSSION THREAD

**Discussion by jeremybradbury** [directus/directus#12472] [Archive]
Currently it's not possible to rename a collection or table. 

I can use a db client to change the table... but then I also need to edit rows in directus_fields, directus_collections, directus_relations etc.

There is absolutely no reason why Directus can/should not allow this, and there's no reason I should be editing (so many) Directus generated meta records myself.

Deleting the table and creating a new one isn't often an option, but that's the easiest way to do it using the admin panel... seems like it shouldn't be.

**Comment by benhaynes** [comment]
Duplicate of : https://github.com/directus/directus/discussions/2711

I renamed that discussion title for clarity.
